pkg/webhook/util/ip.go

```diff
@@ -0,0 +1,83 @@
+package util
```

Comment:
What do you think of adding unit tests for some of these functions?

Reply:
Added unit tests